Easy Build Release.bat

```diff
@@ -0,0 +1,10 @@
+git status
+git fetch
+git submodule update --recursive
```

Comment:
Likewise, submodules should not always be updated automatically, this is a conscious choice. Updating submodules should not be in the build script.